frontends/precinct-scanner/src/screens/scan_warning_screen.test.tsx

```diff
@@ -59,9 +61,9 @@ test('blank ballot', () => {
     adjudicationReasonInfo: [{ type: AdjudicationReason.BlankBallot }],
   });
 
-  screen.getByRole('heading', { name: 'Blank Ballot' });
-  screen.getByText('Your ballot does not have any votes.');
-  userEvent.click(screen.getByRole('button', { name: 'Count blank ballot' }));
+  screen.getByRole('heading', { name: 'No Votes Detected' });
```

Comment:
Nit: keep consistent language between `detected` and `found`. Suggest we simplify header with `No Votes Found`